src/utils/helpers.ts

```diff
@@ -0,0 +1,51 @@
+import { Person } from '../types';
+
+export const sortPeople = (
+  peopleArr: Person[],
+  sort: string | null,
+  order: string | null,
```

Comment:
Consider using an `options` parameter to group the parameters together in the `sortPeople` function, since it has more than 3 parameters. Example: `sortPeople(peopleArr: Person[], options: { sort: string | null, order: string | null })`